packages/zoe/src/contracts/callSpread/fundedCallSpread.js

```diff
@@ -55,7 +55,9 @@ import { Position } from './position.js';
  *   denominator.value=10000)
  */
 
-/** @type {ContractStartFn} */
+/**
+ * @param {ContractFacet<Record<string, any>>} zcf
```

Comment:
Yes, and there is one that treats each custom term access as `unknown`:
```
 * @template {object} [CT=Record<string, unknown>] Contract's custom terms
 * @typedef {Object} ContractFacet
```
The `Record<string, any>` here is only to a more permissive type than the default. I don't think we should put `any` into the default because then you wouldn't get any feedback that things could be better if you defined a type.